src/hipscat_import/margin_cache/margin_cache_arguments.py

```diff
@@ -4,6 +4,9 @@
 import healpy as hp
 import numpy as np
 from hipscat.catalog import Catalog
+from hipscat.catalog.margin_cache.margin_cache_catalog_info import (
+    MarginCacheCatalogInfo,
+)
```

Comment:
This is an odd format, considering that it is only pulling in a single item. But I am guessing this was auto-generated, so it is fine.